jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1350 @@
+          maxLength: 5
+          example: FRPAR
+        facilitySMDGCode:
+          type: string
+          x-stoplight:
+            id: 24m23r848sz38
+          description: "The code used for identifying the specific facility. This code does not include the UN Location Code.\r\n\r\nThe codeList used by SMDG is the [SMDG Terminal Code List](https://smdg.org/wp-content/uploads/Codelists/Terminals/SMDG-Terminal-Code-List-v20210401.xlsx)"
+          maxLength: 6
+          example: ACT
+    UNLocationLocation:
+      title: UNLocationLocation
+      x-stoplight:
+        id: eti9ouqmyta4u
+      type: object
+      description: An interface used to express a location using a `Un Location Code`.
+      required:
+        - UNLocationCode
+      properties:
+        locationName:
+          type: string
+          x-stoplight:
+            id: zlj0kpw923dmh
+          description: The name of the location.
+          maxLength: 100
+          pattern: ^\S+(\s+\S+)*$
+          example: Port of Amsterdam
+        UNLocationCode:
+          type: string
+          x-stoplight:
+            id: et1wvyhg481dq
+          description: The UN Location code specifying where the place is located.
+          maxLength: 5
+          example: FRPAR
+    SubError:
+      title: Sub error
+      x-stoplight:
+        id: w7ekwdiwp8yf6
+      type: object
+      required:
+        - reason
+        - message
+      properties:
+        reason:
+          type: string
+          x-stoplight:
+            id: 5dip777bgtmu4
+          description: High level error message.
+          example: invalidQuery
+        message:
+          type: string
+          x-stoplight:
+            id: n84xm7cqhliyt
+          example: The request did not contain one of the three required query parameters.
+          description: Detailed error message.
+    Timestamp:
+      title: Timestamp
+      x-stoplight:
+        id: jqum2daqmtd0o
+      type: object
+      description: "\t\nA digital record of the time of occurrence of an Operations event."
+      required:
+        - publisher
+        - publisherRole
+        - vesselIMONumber
+        - carrierServiceCode
+        - UNLocationCode
+        - eventClassifierCode
+        - operationsEventTypeCode
+        - eventDateTime
+        - carrierVoyageNumber
+      properties:
+        timestampID:
+          type: string
+          x-stoplight:
+            id: vdcm5gthl1a4a
+          format: uuid
+          example: a82eaf34-cf69-4be3-8df7-881349dde693
+          description: Sender defined ID to enable the receiver to easily detect duplicate messages cased by at least once delivery guarantees.
+        replyToTimestampID:
+          type: string
+          x-stoplight:
+            id: 77n2lh82o6s51
+          format: uuid
+          example: a82eaf34-cf69-4be3-8df7-881349dde693
+          description: Allow sender to respond to a specific timestamp.
```

Comment:
Are these 2 new properties?

Reply:
These properties are there in the schema but not in example.
<img width="524" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/138600841/a89019ed-5528-4952-807b-5995e501271c">